testing_and_setup/compass/ocean/coastal/USDEQU120cr10rr2/build_mesh/config_base_mesh.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<config case="base_mesh">
+
+	<add_executable source="mesh_converter" dest="MpasMeshConverter.x"/>
+	<add_executable source="cell_culler"    dest="MpasCellCuller.x"/>
+
+	<add_link source_path="mpas_model"          source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py"                dest="build_mesh.py"/>
```

Comment:
This will require a new entry in `general.config.ocean`, correct?  Are we sure we want to do this and make the file longer?

Reply:
It is probably better to employ relative paths below, but the challenge there is that if any larger scale movement in COMPASS is made it would break this code.  However, it seems unlikely (at least in the short term) that COMPASS will be completely reorganized.

Reply:
I don't think this requires any additions to `general.config.ocean`, since `mpas_model` has always been in there as far as I know. We used to have an entry for the `jigsaw_to_MPAS` directory, but Mark though that should be taken out. See #201.

Reply:
Good call-- here it is: https://github.com/MPAS-Dev/MPAS-Model/blob/ocean/develop/testing_and_setup/compass/general.config.ocean#L44

Thanks!